Department.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace labs;
+
+class Department
+{
+    public array $employees;
+    public string $name;
+    function __construct($array_of_employee,$name){
```

Comment:
Не соблюдено форматирование, не указаны типы, переменные должны быть в camelCase
```suggestion
    function __construct($array_of_employee,$name){
    function __construct(array $employees, string $name)
    {
```